Makefile

```diff
@@ -302,6 +305,14 @@ run-proxy:
 run-build-vm: $(BUILD_VM) $(BIOS_IMG) $(DEVICETREE_DTB)
 	$(QEMU_SYSTEM) $(QEMU_OPTS) -drive format=qcow2,file=$<
 
+run-parallels-vm:
+	mkdir $(LIVE).hdd
+	qemu-img convert -O parallels $(LIVE).qcow2 $(LIVE).hdd/live.0.$(PARALLELS_UUID).hds
+	qemu-img info -f parallels --output json $(LIVE).hdd/live.0.$(PARALLELS_UUID).hds | jq --raw-output '.["virtual-size"]' | xargs ./tools/parallels_disk.sh $(LIVE) $(PARALLELS_UUID)
+	prlctl create eve --distribution ubuntu --no-hdd --dst $(DIST)/
+	prlctl set eve --device-add hdd --image $(LIVE).hdd --nested-virt on --adaptive-hypervisor on
+	prlctl start eve
```

Comment:
I see that EVE is being launched with video mode set to GUI -- is there any way to use parallels to enable serial console (the way we do it with qemu)?

Reply:
By latest command-line references we able to start in the following modes: 

```
--startup-view <same | window | coherence | fullscreen | modality | headless>
Sets the virtual machine startup view options:
same: Same as the last time
window: Normal window
coherence: Coherence
fullscreen: Full screen
modality: Modality
headless: Headless
```

But we can ssh into VM.

Reply:
I think having the console is important (since ssh may be completely disabled). If there's no way to ask parallels to emulate a serial port on the console -- lets just do a window